### ISSUE TITLE
Feature Request: --slurpfile equivalent for non-json variables

### ISSUE THREAD

**Issue by neokyle1** [stedolan/jq#2210]
While trying to pass an very large non-json string variable into the jq command using the --arg flag, I ended up hitting my OS's max arg size limit. After googling a bit, I saw there was a --argfile/--slurpfile flag that could be used which would fix this problem, however, it appears that these flags expect a json file. I am requesting that another flag be added for passing in the content of a non-json file as a variable.

here is the error when passing in via --arg
```bash
❯ jq -n --arg data "$(dd if=/dev/urandom bs=10000000 count=1 2> /dev/null | strings )" '{data: $data}'
zsh: argument list too long: jq
```

and here is the error when trying to use --slurpfile
```bash
❯ jq -n --slurpfile data <(dd if=/dev/urandom bs=10000000 count=1 2> /dev/null | strings ) '{data: $data}' 
jq: Bad JSON in --slurpfile data /dev/fd/11: Invalid literal at line 2, column 0
```

**Comment by pkoppstein** [comment]
Good news.  --slurpfile is available in jq 1.6

**Comment by neokyle1** [comment]
Actually, it looks like --rawfile is exactly what I am looking for. I found it on the wiki (https://stedolan.github.io/jq/manual/) but it isn't shown on the man page documentation.
